FAQ.md

```diff
@@ -0,0 +1,62 @@
+# FAQs 
+_From emailed responses_
+
+**Q:  Will it be kid friendly?**
+
+A:  Yes! Children under the age of 10 can come for free.  We will have a family camping area, where you can pitch your tent away from the music and late night gatherings. We will also have a family area with activities that young kids can enjoy--hands on creativity.
+
+Many of the activities will also appeal to all ages: from harvesting food for our meals, hiking up the local stream, going to the beach, to live music jams and yoga.
+
+However, you will be expected to be responsible for your children at all times. We won’t have babysitting and you can’t just drop your kids off in the family activities tent. So if you imagine yourself fully participating in workshops, parties, hack-a-thons and more, I think you should plan on bringing someone whose main focus is watching the kids.
+
+**Q:  Will you have a tent big enough for 2 adults, 2 kids, and baby? Or hostel or RV (if able to rent RV)**
+
+A:  Good question!  Our [Glamping tents](https://archive.org) come with one queen bed or two twin cots, so I think it would be a tight squeeze.  Each tent is $400 for the 3-4 nights.
```

Comment:
@whanamura not sure if we want to list this option before confirming tent availability?

Reply:
The MF has 146 of these tents, and their revenue model may depend on some of them getting rented.  In any case, I think we can stop calling them Glamping and describe them as tents that groups can rent, to stop making them seem exclusive.